documentation/slate/source/includes/realtime.md

```diff
@@ -7,14 +7,16 @@ Several endpoints can integrate real time information in their responses. In the
 Real time isn't available on every coverage for Navitia. For real time to be available for a client, it needs to provide real time info about its network to Navitia.
 </aside>
 
-The effect of a disruption can be one of the following:
+The effect of a disruption can be among of the following:
```

Comment:
i think it's 'among the following'

Reply:
Already corrected, @mbounabi called dips on that one :stuck_out_tongue: